diff -ruN vendor/m1/env/src/Parser.php ../Env/src/Parser.php
--- vendor/m1/env/src/Parser.php	2019-08-19 19:53:39.761656370 +0000
+++ ../Env/src/Parser.php	2019-08-21 17:13:25.150491650 +0000
@@ -143,7 +143,9 @@
         foreach ($raw_lines as $raw_line) {
             $this->line_num++;
 
-            if ($this->string_helper->startsWith('#', $raw_line) || !$raw_line) {
+            $line = trim($raw_line);
+
+            if ($this->string_helper->startsWith('#', $line) || !$line) {
                 continue;
             }
 
